MVC/controllers.js

```diff
@@ -1,19 +1,30 @@
-//my callbackfunctions from models
-const findTopics = require('./models')
-const endpoint =require('../endpoints.json')
+const { findTopics, fetchArticleId } = require('./models')
+const endpoint = require('../endpoints.json')
 
 
 const getTopics = function (req, res, next) {
     return findTopics().then((topics) => {
-        //console.log({topics}, "aaaaaaaaaaaaat controls")
         res.status(200).send({ topics })
 
     })
 };
 
-const beOnApi = function(req, res, next){
-    console.log(endpoint)
-    return res.status(200).send({endpoint})  
-}
+const getApiEndPoints = function (req, res, next) {
+    return res.status(200).send({ endpoint })
```

Comment:
as its an array, it makes sense to have this as 'endpoints' plural :)

Reply:
I tried doing that so, and It breaks the test I dont know why